src/OSPSuite.UI/Controls/UxGridView.cs

```diff
@@ -306,14 +308,15 @@ public int OptimalHeight
 
             try
             {
-               int height = viewInfo.CalcRealViewHeight(new Rectangle(0, 0, Int32.MaxValue, Int32.MaxValue));
+               int height = viewInfo.CalcRealViewHeight(new Rectangle(0, 0, MaxValue, MaxValue));
                viewInfo.CalcRealViewHeight(oldBounds);
 
                var scrollInfo = (ScrollInfo) fi.GetValue(this);
                if (scrollInfo.HScrollVisible)
                   height += scrollInfo.HScrollSize;
 
-               return height;
+               //We had this extra height for each row as some is always missing on larger scaling.
+               return height + RowCount * COMPUTED_EXTRA_HEIGHT;
```

Comment:
This was seems to fix it for me. On larger scaling, I need to add some extra pixel per height. This works great in PKSim so I am putting it here. Let see